runconfig/hostconfig_windows.go

```diff
@@ -49,3 +49,10 @@ func ValidateIsolation(hc *container.HostConfig) error {
 func ValidateQoS(hc *container.HostConfig) error {
 	return nil
 }
+
+// ValidateCredentialSpec performs platform specific validation of
+// the credential spec in the hostconfig structure. Linux does not support
+// this Windows-only feature.
+func ValidateCredentialSpec(hc *container.HostConfig) error {
```

Comment:
This is the initial validation where the hostconfig gets passed from the caller into the daemon. The function as you can see is a no-op on Windows. The main point of this function is for similar things  such as `ValidateQoS` or `ValidateIsolation` in `hostconfig_unix.go` to error out if the daemon doesn't support those options. The validation of the value passed is done in `getLibcontainerdCreateOptions` in the update in this PR.